### ISSUE TITLE
[a11y] Cannot trigger the shortcut's menu with talkback gestures

### ISSUE THREAD

**Issue by lobontiumira** [mozilla-mobile/focus-android#7247]
## Steps to reproduce
1. Enable Talkback or Voice Assistant to your device.
2. On Focus' homescreen, add a page as a shortcut.
3. Using TB gestures, long-tap with two fingers on the added shortcut.
4. Access "Rename" or "Remove" options.

### Expected behavior
The user can access either option using TB gestures.

### Actual behavior
The shortcut's menu isn't triggered at all using TB gestures.

### Device information
* Android device: Oppo Reno 6 (Android 12), Samsung Galaxy Note 8 (Android 9)
* Focus version: 6/20 Focus Nightly, Beta 102.0.0-beta.5

**Comment by lobontiumira** [comment]
@mcarare yes, I am able to trigger the context menu on a text, for example, and using the same gestures I am not able to trigger the shortcut's context menu.

**Comment by lobontiumira** [comment]
Following Mihai's suggestion, I am able to trigger the shortcut's menu.
Closing as invalid.
